opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -284,14 +322,19 @@ class Translator {
     /**
      * Internal helper method which makes the actual request to the wink service
      */
-    _makeRequest(path, method, content) {
+    _makeRequest(path, method, content, includeBearerHeader) {
+       
+        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
This seems a bit complicated, why not just check if includeBearerHeader is true below?